TicTacToe/Game.cs

```diff
@@ -119,7 +93,7 @@ public bool IsOver()
         public string DisplayResult()
         {
             string result = "";
-            switch (this.State)
+            switch (State)
             {
                 case "WIN":
                     result = $"Congratulations {this.CurrentPlayer.Name}, you won!";
```

Comment:
To clean up the Game class, you can move these result strings into the CLI class.

Reply:
That makes sense!